cobigen-plugins/cobigen-javaplugin-parent/cobigen-javaplugin/src/test/resources/testdata/unittest/merger/libextension/Anno.java

```diff
@@ -0,0 +1,7 @@
+import javax.annotation.Generated;
+
+@Generated(value= {"com.devonfw.cobigen"}, date = "")
+
+public class Anno{
+
+}
```

Comment:
Is this file still needed?
The generated annotation should only be added to a field or method.